src/devices/ubisys.ts

```diff
@@ -725,6 +726,54 @@ const definitions: Definition[] = [
         },
         ota: ota.ubisys,
     },
+    {
+        // S1-R Series 2 uses the same modelId as the regular S1-R, but the energy clusters are located in endpoint 1 (instead of 4, like the regular S1-R).
+        fingerprint: [
+            {
+                manufacturerName: 'ubisys',
+                modelID: 'S1-R (5601)',
+                endpoints: [{ID: 1, profileID: 260, deviceID: 266, inputClusters: [0, 3, 4, 5, 6, 1794, 2820], outputClusters: []}],
+            },
+        ],
+        model: 'S1-R (Series 2)',
+        vendor: 'ubisys',
+        description: 'Power switch S1-R (Series 2)',
+        extend: [
+            deviceEndpoints({endpoints: {'1': 1, '2': 2, '3': 3, '232': 232}}),
+            identify(),
+            onOff({powerOnBehavior: false}),
+            electricityMeter({cluster: 'metering', configureReporting: false}),
+            commandsOnOff({endpointNames: ['2', '3']}),
+            commandsLevelCtrl({endpointNames: ['2', '3']}),
+            commandsColorCtrl({endpointNames: ['2', '3']}),
+        ],
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['seMetering']);
+            await reporting.readMeteringMultiplierDivisor(endpoint);
+            await reporting.instantaneousDemand(endpoint);
+        },
+        meta: {multiEndpointSkip: ['state', 'power', 'energy']},
+        onEvent: async (type, data, device) => {
+            /*
+             * As per technical doc page 18 section 7.3.4
+             * https://www.ubisys.de/wp-content/uploads/ubisys-s1-technical-reference.pdf
+             *
+             * This cluster uses the binding table for managing command targets.
+             * When factory fresh, this cluster is bound to endpoint #1 to
+             * enable local control.
+             *
+             * We use addBinding to 'record' this default binding.
+             */
+            if (type === 'deviceInterview') {
+                const ep1 = device.getEndpoint(1);
```

Comment:
This should be done in the `configure()`

Reply:
My understanding is that this default binding should only done during the initial device pairing (factory default).